packages/view/backend/sirius-components-view/src/main/resources/model/view.ecore

```diff
@@ -236,11 +236,25 @@
     <eLiterals name="UNSYNCHRONIZED" value="1"/>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="FormDescription" eSuperTypes="#//RepresentationDescription">
+    <eStructuralFeatures xsi:type="ecore:EReference" name="groups" upperBound="-1"
+        eType="#//GroupDescription" containment="true"/>
+  </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="GroupDescription">
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="name" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="labelExpression" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="displayMode" lowerBound="1"
+        eType="#//GroupDisplayMode" defaultValueLiteral="LIST"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="semanticCandidatesExpression"
+        eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString" defaultValueLiteral="aql:self.eContents()"/>
```

Comment:
The `aql:self.eContents()` default is what causes test failures. I think by default we should put `aql:self` to keep the previous behavior and let users opt-in to the possibility of multiple group instances.

Reply:
Done